### PR TITLE
Some small code optimizings

### PR DESCRIPTION
**Pull request by derklaro**

### CODE REVIEW

File: cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java [hunk 1]
```diff
@@ -72,39 +79,33 @@ public static void main(String[] args) throws Exception
...
         if (optionSet.has("debug")) cloudNetLogging.setDebugging(true);
 
         new HeaderFunction();
+        
         CloudNetWrapperConfig cloudNetWrapperConfig = new CloudNetWrapperConfig(cloudNetLogging.getReader());
         CloudNetWrapper cloudNetWrapper = new CloudNetWrapper(optionSet, cloudNetWrapperConfig, cloudNetLogging);
 
-        if (!cloudNetWrapper.bootstrap()) System.exit(0);
+        if (!cloudNetWrapper.bootstrap()) 
+            System.exit(0);
```
[the remark this answers is not in the document]
Comment (derklaro) [reply cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java:101]: Really xD

File: cloudnet-core/src/main/java/de/dytanic/cloudnetcore/bootstrap/CloudBootstrap.java [hunk 2]
```diff
@@ -29,6 +29,12 @@
 
     public static synchronized void main(String[] args) throws Exception
     {
+        if (Float.parseFloat(System.getProperty("java.class.version")) < 52D)
```
Comment (GiantTreeLP) [inline cloudnet-core/src/main/java/de/dytanic/cloudnetcore/bootstrap/CloudBootstrap.java:32]:
```suggestion
        if (Float.parseFloat(System.getProperty("java.class.version")) < 52F) 
```

Either compare Float to Float or Double to Double

File: cloudnet-core/src/main/java/de/dytanic/cloudnetcore/bootstrap/CloudBootstrap.java [hunk 3]
```diff
@@ -29,6 +29,12 @@
 
     public static synchronized void main(String[] args) throws Exception
     {
+        if (Float.parseFloat(System.getProperty("java.class.version")) < 52D) 
+        { 
+            System.out.println("This application needs at least Java 8 or 10.0.1!");
```
Comment (GiantTreeLP) [inline cloudnet-core/src/main/java/de/dytanic/cloudnetcore/bootstrap/CloudBootstrap.java:34]:
```suggestion
            System.out.println("This application needs at least Java 8!"); 
```

This is enough, Java 10 is obsolete now.

File: cloudnet-core/src/main/java/de/dytanic/cloudnetcore/bootstrap/CloudBootstrap.java [hunk 4]
```diff
@@ -132,28 +135,21 @@ public void handleConsole(String input)
...
-                        {
                             System.out.println("Command not found. Use the command \"help\" for further information!");
-                        }
                     }
                 }
-            } catch (Exception ex)
-            {
-
-            }
+            } catch (Exception ex) {}
```
Comment (GiantTreeLP) [inline cloudnet-core/src/main/java/de/dytanic/cloudnetcore/bootstrap/CloudBootstrap.java:149]: I know, this wasn't in the original source, but please log all exceptions.

File: cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/CloudNetWrapperConfig.java [hunk 5]
```diff
@@ -181,14 +173,10 @@ public CloudNetWrapperConfig load()
                 save();
             }
 
-            //generatated configurations
-
             this.proxy_config_host = configuration.getString("general.proxy-config-host");
 
             if (!Files.exists(Paths.get(devServicePath)))
```
Comment (GiantTreeLP) [inline cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/CloudNetWrapperConfig.java:178]:
```suggestion
            if (!Files.exists(Paths.get(devServicePath))) {
                Files.createDirectories(Paths.get(devServicePath));
            }
```

Please keep braces after if-statements

File: cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java [hunk 6]
```diff
@@ -27,6 +27,13 @@
 
     public static void main(String[] args) throws Exception
     {
+        
+        if (Float.parseFloat(System.getProperty("java.class.version")) < 52D)
```
Comment (GiantTreeLP) [inline cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java:31]:
```suggestion
        if (Float.parseFloat(System.getProperty("java.class.version")) < 52F) 
```

Please compare floats to floats

File: cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java [hunk 7]
```diff
@@ -27,6 +27,13 @@
 
     public static void main(String[] args) throws Exception
     {
+        
+        if (Float.parseFloat(System.getProperty("java.class.version")) < 52D) 
+        { 
+            System.out.println("This application needs at least Java 8 or 10.0.1!");
```
Comment (GiantTreeLP) [inline cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java:33]:
```suggestion
            System.out.println("This application needs at least Java 8!"); 
```

This is enough, Java 10 is obsolete.

File: cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java [hunk 8]
```diff
@@ -72,39 +79,33 @@ public static void main(String[] args) throws Exception
             OSSignalBlocker.initSignalCancel();
 
         if (optionSet.has("systemTimer"))
```
Comment (GiantTreeLP) [inline cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java:81]: Please keep the braces after if.

File: cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java [hunk 9]
```diff
@@ -72,39 +79,33 @@ public static void main(String[] args) throws Exception
...
-        {
             new SystemTimer();
-        }
 
         if (optionSet.has("version") || optionSet.has("v"))
-        {
             System.out.println("CloudNet-Wrapper RezSyM Version " + CloudBootstrap.class.getPackage().getImplementationVersion() + "-" + CloudBootstrap.class.getPackage().getSpecificationVersion());
```
Comment (GiantTreeLP) [inline cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java:85]: Please keep the braces after if.

File: cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (GiantTreeLP) [reply cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java:101]: Yes, and please keep braces after if.

File: cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java [hunk 10]
```diff
@@ -72,39 +79,33 @@ public static void main(String[] args) throws Exception
...
         if (!optionSet.has("noconsole"))
         {
             System.out.println("Use the command \"help\" for further information!");
-            String commandLine;
-
-            String user = System.getProperty("user.name");
 
+            final String user = System.getProperty("user.name");
+            String commandLine;
             while (true)
```
Comment (GiantTreeLP) [inline cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java:109]: Analogous to the ifs: Keep braces

File: cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java [hunk 11]
```diff
@@ -123,13 +124,10 @@ public static void main(String[] args) throws Exception
                             ex.printStackTrace();
                         }
                     }
-                } catch (Exception ex)
-                {
-
-                }
+                } catch (Exception ex) {}
```
Comment (GiantTreeLP) [inline cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java:127]: Please log all exceptions

**Review by GiantTreeLP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**